medical/__openerp__.py

```diff
@@ -34,19 +34,16 @@
 #
 ##############################################################################
 {
-
     'name': 'Odoo Medical',
     'version': '1.0',
     'category': 'Medical',
+    'author': "Odoo Medical Team,Odoo Community Association (OCA)",
     'depends': [
         'base',
-        'product'
+        'product',
     ],
-    'author': "Odoo Medical Team,Odoo Community Association (OCA)",
-    'category': 'Generic Modules/Others',
     'application': True,
     'description': """
-
```

Comment:
This is done into #27 I'will improve the README.rst of medical into the next one too